test/subshell/test_runner_test.go

```diff
@@ -32,6 +32,20 @@ func TestMockingRunner(t *testing.T) {
 		must.EqOp(t, "foo called with: bar", res)
 	})
 
+	t.Run("MockCommitMessage", func(t *testing.T) {
+		t.Parallel()
+		runner := subshell.TestRunner{
+			WorkingDir: t.TempDir(),
+			HomeDir:    t.TempDir(),
+			BinDir:     filepath.Join(t.TempDir(), "bin"),
+		}
+
+		runner.MockCommitMessage("test commit message")
+
+		res := runner.MustQuery("bash", "-c", "$GIT_EDITOR output; cat output")
```

Comment:
Rather than shelling out to Bash and then again to `cat`, could this test read the created file directly from Go, for example using `os.ReadFile`? This would also nicely document/test the expected path of the file that is supposed to exist.

Reply:
What does the call `$GIT_EDITOR output` do here?

Reply:
Done in #4453.